app/consulting/index.tsx

```diff
@@ -62,10 +58,13 @@ export default function ConsultingIndex({ tinaProps }) {
   );
 
   useEffect(() => {
-    // as the querystring changes, update the selected tag
-    const qsTag = getSelectedTagFromQuery(params.get("tag"));
-    setSelectedTag(qsTag);
-  }, [params]);
+    // Using Next.js's useSearchParams function leads to complete client-side rendering, which impacts SEO and page load performance,
+    // therefore using javascript's function
```

Comment:
```suggestion
    // We stopped using Next.js's useSearchParams function because it lead to complete client-side rendering, which impacts SEO and page load performance,
    // Therefore we are now using javascript's function
```